teamchains/inputs/hidden_missing_ptk_gen.iced

```diff
@@ -0,0 +1,29 @@
+description: "missing PTK gen; the hidden link skipped one"
+
+users: {
+  "herb": {}
+}
+
+teams: {
+  cabal : {
+    links : [
+      { type: "root", members: { owner: ["herb"] } }
+      { type : "rotate_key" }
+      { type : "rotate_key_hidden" }
+      { type : "rotate_key" }
+      { type : "rotate_key_hidden", corruptors : ptk_gen : 6 }
+      { type : "rotate_key_hidden", corruptors : ptk_gen : 7 }
```

Comment:
does this line need to be there?

Reply:
i think the test still works without it, but i thought i'd check if the missing key wasn't near the endpoint